compute_sdk/tests/unit/test_client.py

```diff
@@ -6,15 +6,21 @@
 import pytest
 from globus_compute_sdk import ContainerSpec, __version__
 from globus_compute_sdk.errors import TaskExecutionFailed
+from globus_compute_sdk.sdk.auth.auth_client import ComputeAuthClient
 from globus_compute_sdk.sdk.login_manager import LoginManager
 from globus_compute_sdk.sdk.utils import get_env_details
 from globus_compute_sdk.sdk.web_client import (
     FunctionRegistrationData,
     FunctionRegistrationMetadata,
+    WebClient,
 )
 from globus_compute_sdk.serialize import ComputeSerializer
 from globus_compute_sdk.serialize.concretes import SELECTABLE_STRATEGIES
 from globus_sdk import __version__ as __version_globus__
+from globus_sdk.experimental.globus_app import UserApp
+from pytest_mock import MockerFixture
+
+_MOCK_BASE = "globus_compute_sdk.sdk.client"
```

Comment:
FWIW, since this is partially new code and not all strictly moved, I observe that most other instances of `_MOCK_BASE`/`_mock_base` include a final period.